app/assets/src/components/views/visualizations/VisualizationsView.jsx

```diff
@@ -0,0 +1,114 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Table } from "~/components/visualizations/table";
+import { merge, pick } from "lodash";
+import GlobeIcon from "~ui/icons/GlobeIcon";
+import LockIcon from "~ui/icons/LockIcon";
+import moment from "moment";
+// TODO (gdingle): refactor with projects_view.scss?
+import cs from "./visualizations_view.scss";
+import cx from "classnames";
+
+class VisualizationsView extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.columns = [
+      {
+        dataKey: "public_access",
+        width: 30,
+        label: "",
+        cellRenderer: this.renderAccess
+      },
+      {
+        dataKey: "details",
+        label: "Visualization",
+        flexGrow: 1,
+        className: cs.detailsCell,
+        cellRenderer: this.renderVisualizationDetails,
+        headerClassName: cs.detailsHeader,
+        sortFunction: p => p.created_at
+      }
+      // TODO: (gdingle): show associated samples and projects
+    ];
+  }
+
+  humanize(key) {
```

Comment:
maybe move this into a utils.js since it appeared in BaseTable as well?

Reply:
done